src/components/carousel/_index.scss

```diff
@@ -95,16 +101,12 @@
 			@include scss.component-properties("carousel-button-additional-next");
 		}
 
-		&--enter-full-screen {
-			@include scss.component-properties("carousel-button-enter-full-screen");
-		}
-
 		&--exit-full-screen {
 			@include scss.component-properties("carousel-button-exit-full-screen");
 		}
 
-		&--toggle-auto-play {
-			@include scss.component-properties("carousel-button-toggle-auto-play");
```

Comment:
Even though we're not using it, it might be worth keeping this token in case customers want to change something specific with the auto play button, and not affect the other buttons.

Reply:
Done, thank you!